task8/TutterApp/src/main/java/models/Post.java

```diff
@@ -0,0 +1,84 @@
+package models;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+public class Post {
+    private Long id;
+    private String description;
+    private Date createdAt;
+    private String author;
+    private String photoLink;
+    private List<String> hashTags;
+    private List<String> likes;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof Post)) return false;
+        Post post = (Post) o;
+        return id.equals(post.id);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id);
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public void setDescription(String description) {
```

Comment:
Traditionally, getters and setters are put together. Not "getters-to-getters" and "setters-to-setters"